x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -14,4 +14,4 @@ export const LIMITED_CONCURRENCY_ENDPOINT_COUNT = 100;
 
 export const TRUSTED_APPS_LIST_API = '/api/endpoint/trusted_apps';
 export const TRUSTED_APPS_CREATE_API = '/api/endpoint/trusted_apps';
```

Comment:
create and list are the same right they should only differ by verb? one is post and the other is get